scss/_footer.scss

```diff
@@ -0,0 +1,166 @@
+.footer .navbar-nav {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-items: flex-start;
+  font-size: $footer-font-size-sm;
+  line-height: $footer-line-height-sm;
+  letter-spacing: $footer-letter-spacing;
+
+  @include media-breakpoint-up(md) {
+    flex-direction: row;
+  }
+
+  .nav-link {
+    display: flex;
+    align-items: center;
+    padding: 0;
+    outline-offset: map-get($spacers, 1);
+
+    &.active::before {
+      content: unset;
+    }
+  }
+}
+
+.footer-heading {
+  font-size: $footer-font-size-md;
+  font-weight: $font-weight-bold;
+  line-height: $footer-line-height-md;
+  letter-spacing: $footer-letter-spacing;
+}
+
+.footer-title-content {
+  padding-top: $footer-mailing-padding-top;
+  padding-bottom: $footer-mailing-padding-bottom;
+
+  @include media-breakpoint-up(md) {
+    padding-bottom: $footer-mailing-padding-bottom-md;
```

Comment:
Shall we rename this variable into something like `$footer-title-content-padding-bottom-md` to be consistant with the name of the class `.footer-title-content` and the name given to that footer's part in the Docs part ?
The problem is the length of the variable name...